test/uk/gov/hmrc/plasticpackagingtax/registration/forms/LiabilityExpectedWeightTest.scala

```diff
@@ -0,0 +1,126 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.plasticpackagingtax.registration.forms
+
+import org.scalatest.matchers.must.Matchers
+import org.scalatest.wordspec.AnyWordSpec
+import play.api.data.FormError
+import uk.gov.hmrc.plasticpackagingtax.registration.forms.LiabilityExpectedWeight.{
+  answer,
+  answerError,
+  maxTotalKg,
+  totalKg,
+  weightBelowThresholdError,
+  weightDecimalError,
+  weightEmptyError,
+  weightFormatError,
+  weightOutOfRangeError,
+  weightThresholdKg
+}
+
+class LiabilityExpectedWeightTest extends AnyWordSpec with Matchers {
+
+  "Liability expect weight validation rules" should {
+
+    "return success" when {
+
+      "is within range of 10 thousand and 100 million" in {
+
+        val input = Map(answer -> "yes", totalKg -> "15000")
+
+        val form = LiabilityExpectedWeight.form().bind(input)
+        form.errors.size mustBe 0
+      }
+
+      "is exactly min allowed amount" in {
+
+        val input = Map(answer -> "yes", totalKg -> weightThresholdKg.toString)
+
+        val form = LiabilityExpectedWeight.form().bind(input)
+        form.errors.size mustBe 0
+      }
+
+      "is exactly max allowed amount" in {
+
+        val input = Map(answer -> "yes", totalKg -> maxTotalKg.toString)
+
+        val form = LiabilityExpectedWeight.form().bind(input)
+        form.errors.size mustBe 0
+      }
```

Comment:
<obs> Maybe you prefer the explicit tests but you could have used a fancy Scalatest Table Drive Property Check test there : https://www.scalatest.org/user_guide/table_driven_property_checks

Reply:
KISS - and this test was derived from LiabilityWeightTest.